Formula/ttygif.rb

```diff
@@ -0,0 +1,18 @@
+class Ttygif < Formula
+  desc "Converts a ttyrec file into gif files"
+  homepage "https://github.com/icholy/ttygif"
+  url "https://github.com/icholy/ttygif/archive/1.3.0.tar.gz"
+  sha256 "18ed967723bacdc1ced6d8ff86bcb3385d940863b8bcb4d3f6b31344f035d7a9"
+
+  depends_on "imagemagick"
+  depends_on "ttyrec"
+
+  def install
+    system "make"
+    bin.install("ttygif")
+  end
+
+  test do
+    system "true"
```

Comment:
We prefer tests that run something substantial. Here you could e.g. create a minimal `ttyrec` file and convert it to a gif.